.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Question of xxx
+about: I want to ask a question about ...
+labels: Question
+---
+
+**General Question**
+
+<!--
+
+Before asking a question, make sure you have:
+
+- Searched questions at https://github.com/infiniflow/infinity
```

Comment:
Replace the link with https://github.com/infiniflow/infinity/issues and https://github.com/infiniflow/infinity/discussions